backend/src/Controller/Main/MainController.php

```diff
@@ -217,4 +224,156 @@ public function deletePackagesAndSubscriptions(): JsonResponse
 
         return $this->json($response);
     }
+
+    /**
+     * store, captain, and supplies: Get complete account status of the user.
```

Comment:
main controller must not have this APIs please move this endpoints to new controller

Reply:
also same for main service